docs/examples.md

```diff
@@ -263,6 +264,44 @@ jobs:
 
 ---
 
+## Limit Runs
+
+This example uses [path filters] to optimize SVGs only if any `.svg` file in the
+repository was changed on `push` events. Even though this Action does nothing
+if a push (or Pull Request) touches no SVGs, you may want the Action to run only
+when an SVG has actually changed.
+
+> **Warning** This will cause the entire Workflow to be run only when an SVG
+> changes. Jobs that should run for every push or Pull Request must be specified
+> in a separate Workflow file.
+
+```yml
+on:
```

Comment:
This snippet is missing the first few lines when compared to the other examples

```suggestion
```yml
# .github/workflows/optimize.yml

name: Optimize
on:
```